internal/cmd/login_test.go

```diff
@@ -93,21 +119,11 @@ func TestLoginCmd(t *testing.T) {
 		ctx, cancel := context.WithCancel(ctx)
 		t.Cleanup(cancel)
 
-		console := newConsole(t)
-		ctx = PatchCLIWithPTY(ctx, console.Tty())
-
-		g, ctx := errgroup.WithContext(ctx)
-		g.Go(func() error {
-			// TODO: remove --skip-tls-verify
-			return Run(ctx, "login", srv.Addrs.HTTPS.String(), "--skip-tls-verify", "--no-agent", "--key", adminAccessKey)
-		})
-
-		exp := expector{console: console}
-		exp.ExpectString(t, "Logged in as")
-
-		assert.NilError(t, g.Wait())
+		// TODO: remove --skip-tls-verify
+		err := Run(ctx, "login", srv.Addrs.HTTPS.String(), "--skip-tls-verify", "--no-agent", "--key", adminAccessKey)
+		assert.NilError(t, err)
```

Comment:
Since we are only testing that the agent is not started we don't need the console and goroutine in this case. We can run the full command and check the error.